modules/google-maps/src/utils.js

```diff
@@ -223,7 +223,8 @@ function getEventPixel(event, deck) {
   const point = deck.getViewports()[0].project([event.latLng.lng(), event.latLng.lat()]);
   return {
     x: point[0],
-    y: point[1]
+    y: point[1],
+    radius: deck.props.pickingRadius
```

Comment:
This is going to break the typing of this object. Instead of attaching radius to the point, try applying it when picking is performed:

`deck.pickObject({...mockEvent.offsetCenter, radius})`

Reply:
Gotcha. I'm unsure, should it only be in the case 'click' event, or in the mockEvent passed to all the evengs?